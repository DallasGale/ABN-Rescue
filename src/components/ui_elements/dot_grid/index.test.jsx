/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import AllDots from '.'

describe('<AllDots />', () => {
  it('renders without crashing', () => {
    const component = render(<AllDots />)
    expect(component).toMatchSnapshot()
  })
})
