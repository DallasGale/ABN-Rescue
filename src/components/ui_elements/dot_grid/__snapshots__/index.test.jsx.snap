// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AllDots /> renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styled__StyledGrid-z2dehs-1 dggVZi"
      >
        <div
          class="styled__StyledDotRow-z2dehs-3 ctOLyk"
        >
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
          <div
            class="appearing styled__StyledDot-z2dehs-2 eEunID"
          >
            <svg
              class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
              viewBox="0 0 3623 3623"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
                color="#00AD20"
                cx="1811.5"
                cy="1811.5"
                r="1811.5"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styled__StyledGrid-z2dehs-1 dggVZi"
    >
      <div
        class="styled__StyledDotRow-z2dehs-3 ctOLyk"
      >
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
        <div
          class="appearing styled__StyledDot-z2dehs-2 eEunID"
        >
          <svg
            class="filled-circle__Svg-sc-1gwuu5l-0 gHjgAd"
            viewBox="0 0 3623 3623"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="filled-circle__StyledCircle-sc-1gwuu5l-1 cOEXBT"
              color="#00AD20"
              cx="1811.5"
              cy="1811.5"
              r="1811.5"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
